widgets/src/main/resources/org/pentaho/gwt/widgets/themes/public/themes/ruby/globalRuby.css

```diff
@@ -8693,6 +8693,22 @@ div.dijitDateTextBox.dijitTextBox .dijitReset.dijitRight.dijitButtonNode.dijitAr
   border-bottom: 1px solid #e0e0e0;
 }
 
+:where(.responsive#general-datasource-window) #sql-label {
+  white-space: break-spaces !important;
+}
```

Comment:
On `GwtLabel` we have the call to `setWordWrap` [commented out](https://github.com/pentaho/pentaho-commons-xul/blob/wcag/gwt/impl/src/main/java/org/pentaho/ui/xul/gwt/tags/GwtLabel.java#L65).

In a previous talk with @Pancho7, we discussed the idea of introducing a prop here
to control the wrap boolean value, that would enable us to properly set the `white-space` style for this labels

@dcleao do you think this would be a good solution to remove the need of this css override?

Reply:
You have a point. In my mind, `multiline` was tied to different markup, but that's only the case of `pre`, after all, and the only current impact of `multiline` is to choose between the `white-space` values of `normal` and `nowrap`.

Looking around, it seems that `<label multiline=true` is used mostly to display messages (error, warning, info, help, ...). Labels associated with input fields don't have multiline specified, defaulting to false.

The existing CSS rules referenced above are really for the case of being `multiline` and `pre` (as they select the `.label-scroll-panel` class only existent when `pre`).

For labels of input fields, though, in responsive mode, I think their text better break by default... Do you agree? 

If so, similarly to what was done with other properties (flex, spacing), we could have different _defaults_ for `multiline` depending on being in responsive mode or not. Responsive mode is enabled by a dialog, but then only made effective by a responsive theme.

This is one way it could be done:
1. Define `GwtLabel#isMultilineSpecified()` and field `isMultilineSpecified`, defaulting to `false`.
2. When `setMultiline( . )` is called, set `isMultilineSpecified` to `true` (cannot be reset..).
3. In the `layout` method, if `isMultilineSpecified` is `true`, then add a style name of `multiline-specified`, otherwise, remove it.
4. In CSS, create rules that assign the corresponding _default value_ of `word-break`.
```css
:where(.responsive.pentaho-xul-gwt) .xul-label:not(.multiline-specified) {
  word-break: nowrap !important;
}
:where(.responsive.pentaho-xul-gwt) .xul-label:not(.multiline-specified).multiline {
  word-break: normal !important;
}
```

To force `nowrap` in responsive mode, the XUL attribute `multiline="false"` would be specified.

Reply:
While talking offline, we realized the above rules should be, instead:
```css
:where(.responsive.pentaho-xul-gwt) .xul-label:not(.multiline-specified) {
  white-space: normal !important;
}
```

A single rule is needed. Plus replaced incorrect use of `word-break` by `white-space`...

Reply:
Done 👍